entity-registry/src/main/java/com/linkedin/metadata/models/DefaultEventSpec.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.metadata.models;
+
+import com.linkedin.data.schema.RecordDataSchema;
+import com.linkedin.metadata.models.annotation.EventAnnotation;
+import javax.annotation.Nonnull;
+
+
+public class DefaultEventSpec implements EventSpec {
```

Comment:
We can do something like below to remove boilerplate code. (Note you need to remove _ from variable name to make it work)
```
@RequiredArgsConstructor
@Getter
public class DefaultEventSpec implements EventSpec {
  private final String name;
  private final EventAnnotation eventAnnotation;
  private final RecordDataSchema pegasusSchema;
}
```